nx.json

```diff
@@ -28,7 +28,7 @@
         "{projectRoot}/test/**",
         "{projectRoot}/types.ts"
       ],
-      "dependsOn": ["build"],
+      "dependsOn": [],
```

Comment:
I will make a follow-up PR to sort out some caching issues. Unless you're executing tasks with `pnpm lerna run test --scope=@turf/pkg` this file doesn't come into play though, and CI doesn't cache anything so we're safe there as well.

Reply:
https://github.com/Turfjs/turf/pull/2685 is the change to the `inputs` to fix test caching. I intentionally didn't remove the `build` dependsOn entry because you're doing that here and I didn't want to conflict.